src/components/ButtonDropdownSelect/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {useState} from 'react';
+import { Box, ListItemIcon, ListItemText, Menu, MenuItem, MenuList, Typography } from '@mui/material';
+import { DropdownItem } from '../types';
+import { IconName } from '../Icon/icons';
+import Button from '../Button/Button';
+import Icon from '../Icon/Icon';
+import './styles.scss';
+
+type ButtonDropdownSelectProps = {
+  id: string;
+  options: DropdownItem[];
+  initialSelection: DropdownItem[];
+  onChange: (selection: DropdownItem[]) => void;
```

Comment:
Values alone should be enough? Dropdown items have more semantics (rendered label etc..)